README.md

```diff
@@ -50,18 +46,18 @@ $ gem install slayer
 
 ### Commands
 
-Slayer Commands should implement `call`, which will `pass` or `fail` the service based on input. Commands return a `Slayer::Result` which has a predictable interface for determining `success?` or `failure?`, a 'value' payload object, a 'status' value, and a user presentable `message`.
+Slayer Commands should implement `call`, which will `pass` or `fail` the service based on input. Commands return a `Slayer::Result` which has a predictable interface for determining `passed?` or `failed?`, a 'value' payload object, a 'status' value, and a user presentable `message`.
 
 ```ruby
 # A Command that passes when given the string "foo"
 # and fails if given anything else.
 class FooCommand < Slayer::Command
   def call(foo:)
     unless foo == "foo"
-      flunk! value: foo, message: "Argument must be foo!"
+      ko! value: foo, message: "Argument must be foo!"
     end
 
-    pass! value: foo
+    ok value: foo
```

Comment:
`ok!` is weird because in order to stop execution it has to throw an exception. We can do that, but it's... kinda gross?

`ko`  does exist, and I want to update the docs to recommend it over `ko!` — with the Python version of this we went down a path of "always return your result, don't throw errors" and I liked that better. I need to come through and mark the old methods as deprecated, and I think we should do the same for `ko!` and favor `return ko`.

Reply:
I'm also totally down with removing the angry version of things. Just return everything is fantastic.